### DISCUSSION TITLE
Call external tool for proprietary file formats

### DISCUSSION THREAD

**Discussion by kafunamapa** [WinMerge/winmerge#2553] [Ideas]
We use NI TestStand for some of our projects and the *.seq files are in a proprietary format. When we diff a project folder, WinMerge will detect that the *.seq binary files are different, but the diff view is meaningless. NI provides a diff/marge tool for this filetype. I'd like to be able to launch an external diff tool depending on file extension.

**Comment by sdottaka** [comment]
Unfortunately, WinMerge does not currently have the ability to register external comparison tools.
However, if you select Both Shell menu in the context menu of the Folder Comparison window, the Explorer shell menu will be displayed and you will be able to select the Send To menu.
If you register the tool you want to launch (such as NI TestStand) in Send To, you will be able to use it from WinMerge.
![image](https://github.com/user-attachments/assets/507e3ea8-169e-4d62-bbdf-bf4721843525)
![image](https://github.com/user-attachments/assets/e3a05f7b-65a7-4ca0-89a0-d527dd912de7)
